packages/webpack-build-scripts/index.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the terms of the LICENSE file distributed with this project.
+ */
+
+const baseConfig = require("./webpack.config.base");
+
+module.exports = {
+    baseConfig,
+
+    COMMON_EXTERNALS: {
```

Comment:
easy enough to fix #1805 while we're at it?

Reply:
not really, i'd like to do that in a separate PR